test/components/form/submission.spec.js

```diff
@@ -0,0 +1,51 @@
+import testData from '../../data';
+import { load } from '../../util/http';
+import { mockLogin } from '../../util/session';
+import simpleXml from '../../data/simple';
+
+describe('FormSubmission', () => {
+  beforeEach(() => {
+    mockLogin();
+  });
+
+  it('sends the correct initial requests - Web Forms', () => {
+    testData.extendedForms.createPast(1, { xmlFormId: 'a', webformsEnabled: true });
+    return load('/projects/1/forms/a/submissions/new')
+      .respondWithData(() => simpleXml)
+      .testRequests([
+        { url: '/v1/projects/1/forms/a' },
+        { url: '/v1/projects/1/forms/a.xml' },
+      ]);
+  });
+
+  it('sends the correct initial requests - Enketo', () => {
+    testData.extendedForms.createPast(1, { xmlFormId: 'a' });
+    return load('/f/enketo-id/new?st=token')
+      .testRequests([
+        { url: '/v1/enketo-ids/enketo-id/form?st=token' }
+      ]);
+  });
```

Comment:
How about adding one or more tests that showcase the requests to /v1/test/... endpoints?